jscomp/others/dom.ml

```diff
@@ -14,6 +14,21 @@ type cssStyleSheet
 (* events (early) *)
 type 'a eventTarget_like
 type eventTarget = _baseClass eventTarget_like
+type _messagePort
+type messagePort = _messagePort eventTarget_like
+type _serviceWorker
+type serviceWorker = _serviceWorker eventTarget_like
+type _worker
+type worker = _worker eventTarget_like
+type 'a _workerGlobalScope
+type 'a workerGlobalScope_like = 'a _workerGlobalScope eventTarget_like
+type workerGlobalScope = _baseClass workerGlobalScope_like
+type _dedicatedWorkerGlobalScope
+type dedicatedWorkerGlobalScope = _dedicatedWorkerGlobalScope workerGlobalScope_like
+type _serviceWorkerGlobalScope
+type serviceWorkerGlobalScope = _serviceWorkerGlobalScope workerGlobalScope_like
+type _sharedWorkerGlobalScope
+type sharedWorkerGlobalScope = _sharedWorkerGlobalScope workerGlobalScope_like
```

Comment:
@anmonteiro Hi, I add these types since your last review. Could you please take a look at it?